front/scripts/uploadController.js

```diff
@@ -63,6 +58,23 @@ $(() => {
     $('#chapterSubmit').on('click', () => {
       uploadController.readFileInput($('#chapter1FileInput'), uploadController.converter);
     });
+    $('#chapterSubmit').on('click', () => {
+      uploadController.staticData = new uploadController.Static(
+        $('#isbnInput').val(),
+        $('#authorInput').val(),
+        $('#bookTitle').val(),
+        $('#emailIntroInput').val()
+      );
+      if ($('#bannerInput').val()) {
```

Comment:
We'll have our if block in the handlebars template since if the value winds up being an empty string, that will evaluate to a falsey boolean.